README.md

```diff
@@ -16,28 +17,50 @@ This allows messaging clients to verify the identity of
 users automatically, and prevents malicious/compromised servers from
 hijacking secure communications without getting caught.
 
+This repository provides a [Golang](https://golang.org) implementation of the 
+CONIKS system. The implementation consists of a library, described in the 
+following section, and a standalone CONIKS-server and -client using this library.
+
 ## Golang Library
+
 The packages in this library implement the various components of the CONIKS
 system and may be imported individually.
 
-- ``bots``: Registration proxies for user account verification
-- ``client``: A reference implementation of a CONIKS client
-- ``crypto``: Cryptographic algorithms and operations
-- ``keyserver``: A reference implementation of a CONIKS key server
-- ``merkletree``: Merkle prefix tree and related data structures
-- ``utils``: Utility functions
-- ``protocol``: CONIKS protocols implementation/library
-- ``storage``: Hooks for persistent storage backend (currently unused)
+- `bots`: Registration proxies for user account verification
+- `client`: A reference implementation of a CONIKS client
+- `crypto`: Cryptographic algorithms and operations
+- `keyserver`: A reference implementation of a CONIKS key server
+- `merkletree`: Merkle prefix tree and related data structures
+- `utils`: Utility functions
+- `protocol`: CONIKS protocols implementation/library
+- `storage`: Hooks for persistent storage backend (currently unused)
 
 ## Disclaimer
+
 Please keep in mind that this CONIKS library is under active development.
 The repository may contain experimental features that aren't fully tested.
 We recommend using a [tagged release](https://github.com/coniks-sys/coniks-go/releases).
 
-##Documentation
+## Installation
+
+The preferred way to install CONIKS reference server or client implementation
+is by downloading the [latest release](releases/latest).
+<!-- and by following the contained instructions. -->
+
+If you prefer to build and install from source you need to have a recent 
+[Golang](https://golang.org/doc/install) version installed (1.6+). 
+If Golang is setup correctly, you can simply run `go get github.com/coniks-sys/coniks-go` in your command line.
+
+In both cases, installing CONIKS installs a [CONIKS-server](keyserver/) and a
+simple command-line [client](client). Both commands are documented in their
+own package ([server here](keyserver) and [client here](client)).
```

Comment:
I would change the last sentence to something like "For usage instructions, see the documentation in their respective packages (server here and client here)". Should we mention the bots here, too?